instructure/canvas-ios
# Offline quizzes

refs: MBL-16768
affects: Student
release note: none 
test plan: 
1. Select quizzes when syncing offline content 
2. Turn off internet access
3. Open Quizzes tab
4. You should be able to see the quiz list 
5. On the details page, the `Take quiz`, `Resume quiz`, `View result` button should be disabled. 


## Checklist
